CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
-# Contributing to Real-time Ubuntu
+# Contributing to Real-time Ubuntu docs
+
+The [Real-time Ubuntu documentation] lives in the `docs/` directory and is deployed on Read the Docs.
+It is written in [reStructuredText] and built with [Sphinx].
+
+The documentation should follow the [Canonical documentation style guide] in use for documentation at Canonical.
+
+To install everything you need, go to the `docs/` directory and type:
+
+```shell
+make install
+```
+
+This will create a virtual environment with all the tooling dedicated to validate and render the documentation.
+
+To get a live preview and validation of the documentation, you can then run:
+
+```shell
+make run
+```
+
+This will provide a link to a locally rendered version of the documentation that will be updated every time a file is modified.
```

Comment:
```suggestion
This will provide a locally rendered version of the documentation at `http://127.0.0.1:8000` that will be updated every time a file is modified.
```